libs/checkpoint-postgres/langgraph/checkpoint/postgres/__init__.py

```diff
@@ -180,6 +180,69 @@ def list(
                     self._load_writes(value["pending_writes"]),
                 )
 
+    def get_writes(self, task_id: str, ttl: int = None) -> List[Any]:
```

Comment:
we should accept a list of task ids ideally, as when we do these queries we might have multiple tasks to get cached writes for (and will be more efficient to run a single query for all)

Reply:
added